frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -18,10 +26,179 @@ function ReaderCoptListener:onReadSettings(config)
 
     local status_line = config:readSetting("copt_status_line") or G_reader_settings:readSetting("copt_status_line") or 1
     self.ui:handleEvent(Event:new("SetStatusLine", status_line, true))
```

Comment:
It feels that logically, this should be done after setting all the new properties: You set how it shoud look if enabled, you then set if enabled or not, and then you send this event. (Even if may be it doesn't matter technically.)